Models/Interior/Panel/Instruments/kx165/kx165-1.xml

```diff
@@ -12,7 +12,7 @@
         flightgear sessions
 
         aircraft.data.add(
-        "instrumentation/comm[0]/volume",
+        "instrumentation/comm[0]/volume-selected",
```

Comment:
volume-selected was chosen based on similar work done in c182s.

Reply:
:+1: